dataeng/resources/event-export-incremental.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+####################################################################
+# Exporter configuration
+
+ROOT=${WORKSPACE}/analytics-secure/analytics-exporter
+SECURE_HASH=`GIT_DIR=./analytics-secure/.git git rev-parse HEAD`
+EXPORTER_CONFIG_BUCKET=s3://edx-analytics-scratch/exporter/config/$SECURE_HASH
+EXPORTER_CONFIG_PATH=${EXPORTER_CONFIG_BUCKET}/${EXPORTER_CONFIG}
+GPG_KEYS_PATH=${EXPORTER_CONFIG_BUCKET}/gpg-keys
+
+####################################################################
+# Upload configuration files to s3
+
+pip install awscli
+
+echo ${WORKSPACE}/data-czar-keys/${EXPORTER_CONFIG}
+
+aws s3 cp ${WORKSPACE}/data-czar-keys/${EXPORTER_CONFIG} ${EXPORTER_CONFIG_PATH}
+aws s3 cp --recursive ${WORKSPACE}/data-czar-keys/ ${GPG_KEYS_PATH}
+
+####################################################################
+# Run exporter task
+
+if [ "x$NUM_REDUCE_TASKS" = "x" ]; then
```

Comment:
Also update this script with `[ -z "$NUM_REDUCE_TASKS" ]` and `${WORKSPACE}/...` like the rest.